.github/workflows/docs.yml

```
@@ -0,0 +1,72 @@
+
+      - name: Install dependencies
+        run: |
+          poetry install --with dev
+
+      - name: Build documentation
+        run: |
+          cd docs
+          poetry run sphinx-build -b html . _build/html
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
```

actionlint (1.7.9)

the runner of "actions/configure-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)